elk/__main__.py

```diff
@@ -1,12 +1,57 @@
 """Main entry point for `elk`."""
 
-from .argparsers import get_extraction_parser, get_training_parser
-from .files import args_to_uuid
-from .list import list_runs
-from argparse import ArgumentParser
-from contextlib import nullcontext, redirect_stdout
 import logging
 import os
+from argparse import ArgumentParser
+from contextlib import nullcontext, redirect_stdout
+
+from elk.evaluate.evaluate import evaluate
+from elk.files import args_to_uuid, elk_cache_dir
+from elk.list import list_runs
+
+from .argparsers import get_extraction_parser, get_training_parser, get_evaluate_parser
+from .extraction.extraction_main import run as run_extraction
+from .training.train import train
+
+
+# TODO: Move function to a better place...
+def sweep(args):
```

Comment:
Handling everything via the command line / args makes the sweep a bit annoying. I guess it might be better to have a .yaml file for the config... 

elk sweep burns.yaml